index.js

```diff
@@ -94,23 +122,38 @@ function addPath (path) {
   }
 }
 
+/**
+ * Register multiple aliases.
+ * @param {{string: string}} aliases
+ * @example
+ * ModuleAlias.addAliases({
+ * 'Components': '/src/components',
+ * 'Utils': '/src/utils'
+ * })
+ */
 function addAliases (aliases) {
   for (var alias in aliases) {
     addAlias(alias, aliases[alias])
   }
 }
 
-function addAlias (alias, target) {
-  moduleAliases[alias] = target
+/**
+ * Register a single alias.
+ * @param {string} alias Alias
+ * @param {string} path Target path
+ * @example ModuleAlias.addAlias('Utils', '/src/utils')
```

Comment:
Changing to lower case makes sense because `moduleAlias` is not a class

```suggestion
 * @example moduleAlias.addAlias('utils', path.join(__dirname, 'src/utils'))
```